G5/g5-client-app-creator
# Spinup apps with specified dynos and AWS deploys by default

Testing
 * Create a new client with t1-client-app-creator at master: cms and cls spun up with $25 production dynos
 * Deploy this change and create a new client: cms has $25 dyno, cls has $7 dyno, and cms has new expected env vars set as expected (BITBUCKET_USER, PROVISIONER_URL)

When deployed these env vars need to be set on g5-client-app-creator: BITBUCKET_USER, PROVISIONER_URL, ID_RSA (new key that accesses bitbucket, heroku, and github)
